test/hooks.js

```diff
@@ -39,6 +39,19 @@ test('hooks can be async', async t => {
 	await server.close();
 });
 
+test('hooks can be empty object', async t => {
+	const server = await createTestServer();
+	server.get('/', (request, response) => {
+		response.end('empty hook');
```

Comment:
Make `'empty hook'` as variable and use it in the assertion too. It's a good practice to do this to ensure there are no typos.